game-core/src/main/java/games/strategy/engine/lobby/client/login/LobbyServerProperties.java

```diff
@@ -38,9 +44,9 @@
   @Nonnull
   private final Integer port;
 
-  /** URI for the http lobby server. */
+  /** The port the https lobby server is listening on. */
   @Nonnull
-  private final URI httpServerUri;
+  private final Integer httpsPort;
```

Comment:
I think we had a discussion about this before.
The only reason this is an Integer instead of an int is because the annotation to make it required in the builder right? Maybe I'm overanalyzing this.
In any case doesn't using just int work here, because this field is final?

Reply:
> The only reason this is an Integer instead of an int is because the annotation to make it required in the builder right?

Yes, otherwise it defaults to zero when not specified. The point of the `@Nonnull` is to require a person to specify the value.